features/projects/create_task.feature

```diff
@@ -0,0 +1,21 @@
+@api
+Feature: Creating a task
+
+  Rules:
+    - Project id is required
+    - Name of task is required
+    - Description of task is required
```

Comment:
If there are specific rules you want to ensure are enforced, it would probably be a good idea to write some scenarios showing the rules are followed. Perhaps 3 failure scenarios showing if you leave out any of those items you receive an appropriate error message.